src/xml2js.coffee

```diff
@@ -169,6 +169,8 @@ class exports.Builder
             if typeof child is 'string' && @options.cdata && requiresCDATA child
               element = element.ele(key).raw(wrapCDATA child).up()
             else
+              if child === undefined || child === null
```

Comment:
```child?``` isn't the same as ```child is null```, ```child``` checks to see if it isn't null. We want to check if it is null.

Changing that line to ```child?``` blows up a lot of tests